### PR TITLE
Update issue templates 

### PR DESCRIPTION
**Pull request by kelea99**
Created an issue template to test, in an effort to give essential information in streamlined way to IT staff working on issue.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/kel-issue-template.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+### Impact 
+_Please include hard deadlines, if the exhibit is part of an event, the issue is stopping work, etc._
+
+### Priority recommendation
+
+- [ ] asap
+- [ ] within the next 3 weeks
+- [ ] PO will prioritize
+
+## Sudden Priority Justification
```
Comment (tpendragon) [inline .github/ISSUE_TEMPLATE/kel-issue-template.md:21]: This header's only required if asap or "within the next 3 weeks" is checked.

Comment (kelea99) [reply .github/ISSUE_TEMPLATE/kel-issue-template.md:21]: Would you like to keep that header in (with descrip), or would you prefer to add it if needed?

Comment (tpendragon) [reply .github/ISSUE_TEMPLATE/kel-issue-template.md:21]: Whatever you'd prefer, I thought you might just want a note so you don't feel like you have to always add it.

Comment (kelea99) [reply .github/ISSUE_TEMPLATE/kel-issue-template.md:21]: Ill keep it with the descrip. it is really helpful

**Review by tpendragon — APPROVED** [review]

**Review by hackartisan — APPROVED** [review]
Looks good to me. I looked at the documentation but I'm still not exactly sure how this integrates once we merge it. I guess we'll find out!

**Comment by kelea99** [comment]
I look forward to finding out!!

**Pull request merged**